pyproject.toml

```diff
@@ -107,7 +107,7 @@ packages = ["nilearn"]
 source = "vcs"
 
 [tool.pytest.ini_options]
-addopts = "-ra --strict-config --strict-markers --doctest-modules --showlocals -s -vv --durations=0 --template=maint_tools/templates/index.html -n auto"
```

Comment:
remove parallel testing from the default arguments of pytest and only use it in CI via tox (would otherwise interfere with pytest options like `-x --f -s`